go/teams/transactions.go

```diff
@@ -836,7 +836,7 @@ func (tx *AddMemberTx) Post(mctx libkb.MetaContext) (err error) {
 		return err
 	}
 
-	team.notify(mctx.Ctx(), keybase1.TeamChangeSet{MembershipChanged: true})
+	team.notify(mctx.Ctx(), keybase1.TeamChangeSet{MembershipChanged: true}, nextSeqno)
```

Comment:
nextSeqno-1 ? next vs last

Reply:
ah good call, thanks. this was the particularly buggy bit.

Reply:
...since it could result in stale values being used...